game/player.cc

```diff
@@ -79,13 +77,10 @@ void Player::calcRowRank() {
 			m_prevLineScore /= m_maxLineScore;
 			m_feedbackFader.setValue(1.0);
 		} else {
-			m_prevLineScore = -1;
+			m_prevLineScore = -1.0;
 		}
 		m_lineScore = 0;
 	}
 }
 
-PlayerItem::PlayerItem(PlayerId id) 
-: id(id)
-{
-}
+PlayerItem::PlayerItem(PlayerId _id) { if (_id) id = _id.value(); }
```

Comment:
```suggestion
PlayerItem::PlayerItem(PlayerId _id) : id{_id} {}
```

Reply:
This one won't work because PlayerItem actually wants an unsigned and not an optional.